packages/playwright-core/src/client/network.ts

```diff
@@ -141,7 +141,7 @@ export class Request extends ChannelOwner<channels.RequestChannel> implements ap
       return null;
 
     const contentType = this.headers()['content-type'];
-    if (contentType === 'application/x-www-form-urlencoded') {
```

Comment:
Should we also support more exotic `charset=UTF-8; application/x-www-form-urlencoded` ?